packages/relay/src/index.ts

```diff
@@ -29,6 +29,8 @@ main().catch((err) => {
     process.exit(1)
 })
 
+export let socketManager: SocketManager
```

Comment:
I added one object in `SocketManager`, maybe we could use one of them, remove another.